File: src/MixNetFx/FacessoData/DataClasses/ParameterClasses/WorkgroupBaseDataParameters.vb
Public Class WorkgroupBaseDataPrintParameters

    Private myPrintWorkgroups As Boolean
    Private myPrintAssignedLabourValues As Boolean
    Private myPrintShiftTimes As Boolean
    Private myPrintProductivityHistory As Integer
    Private myOnlyPrintListOfLabourValues As Boolean

    Public Property PrintWorkgroups() As Boolean
        Get
            Return myPrintWorkgroups
        End Get
        Set(ByVal value As Boolean)
            myPrintWorkgroups = value
        End Set
    End Property

    Public Property PrintAssignedLabourValues() As Boolean
        Get
            Return myPrintAssignedLabourValues
        End Get
        Set(ByVal value As Boolean)
            myPrintAssignedLabourValues = value
        End Set
    End Property

    Public Property PrintShiftTimes() As Boolean
        Get
            Return myPrintShiftTimes
        End Get
        Set(ByVal value As Boolean)
            myPrintShiftTimes = value
        End Set
    End Property

    Public Property VisualizeProductivityHistory() As Integer
        Get
            Return myPrintProductivityHistory
        End Get
        Set(ByVal value As Integer)
            myPrintProductivityHistory = value
        End Set
    End Property

    Public Property OnlyPrintListOfLabourValues() As Boolean
        Get
            Return myOnlyPrintListOfLabourValues
        End Get
        Set(ByVal value As Boolean)
            myOnlyPrintListOfLabourValues = value
        End Set
    End Property
End Class

File: src/MixNetFx/FacessoData/DataClasses/ParameterClasses/WorkgroupBaseDataParameters.cs
namespace Facesso.Data
{
    public class WorkgroupBaseDataPrintParameters
    {
        private bool myPrintWorkgroups;
        private bool myPrintAssignedLabourValues;
        private bool myPrintShiftTimes;
        private int myPrintProductivityHistory;
        private bool myOnlyPrintListOfLabourValues;

        public bool PrintWorkgroups
        {
            get { return myPrintWorkgroups; }
            set { myPrintWorkgroups = value; }
        }

        public bool PrintAssignedLabourValues
        {
            get { return myPrintAssignedLabourValues; }
            set { myPrintAssignedLabourValues = value; }
        }

        public bool PrintShiftTimes
        {
            get { return myPrintShiftTimes; }
            set { myPrintShiftTimes = value; }
        }

        public int VisualizeProductivityHistory
        {
            get { return myPrintProductivityHistory; }
            set { myPrintProductivityHistory = value; }
        }

        public bool OnlyPrintListOfLabourValues
        {
            get { return myOnlyPrintListOfLabourValues; }
            set { myOnlyPrintListOfLabourValues = value; }
        }
    }
}
